cl/opinion_page/tests.py

```diff
@@ -13,6 +20,7 @@
 from cl.lib.test_helpers import SitemapTest
 from cl.opinion_page.views import make_docket_title
 from cl.search.models import Citation, Docket
+from cl.people_db.models import Person
```

Comment:
Similarly, using PyCharm's import tools will alphabetize new imports properly. Can you bump this up a line?

Reply:
√